hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/utils/db/TestRDBTableStore.java

```diff
@@ -428,9 +428,10 @@ public void testIteratorRemoveFromDB() throws Exception {
     // Remove without next removes first entry.
     try (Table<byte[], byte[]> testTable = rdbStore.getTable("Fifth")) {
       writeToTable(testTable, 3);
-      TableIterator<byte[], ? extends Table.KeyValue<byte[], byte[]>> iterator =
-          testTable.iterator();
-      iterator.removeFromDB();
+      try (TableIterator<?, ? extends Table.KeyValue<?, ?>> iterator =
```

Comment:
Just curious. Why does this patch replace many explicit type declarations with wildcards?